### ISSUE TITLE
Version Assemblies - EOL Node version

### ISSUE THREAD

**Issue by kspanier2013** [colindembovsky/cols-agent-tasks#211]
Hello there.

Today I noticed the following warning on Azure DevOps build pipelines that reference the **VersionAssemblies@2** task, during the **Initialize job** build step:

```
Downloading task: VersionAssemblies (2.2.178)
##[warning]Task 'Version Assemblies' version 2 (VersionAssemblies@2) is dependent on a Node version (6) that is end-of-life. Contact the extension owner for an updated version of the task. Task maintainers should review Node upgrade guidance: https://aka.ms/node-runner-guidance
```

If it matters, the runner of the build pipeline was an azure-hosted windows-2022 image, version 20241031.1.0.

No tangible issues have been noticed other than this warning.

**Comment by o-o00o-o** [comment]
Yes, we have been seeing this too. It would be good to understand if this will be resolved or if we need to migrate to a different task

**Comment by aligneddev** [comment]
has this been taken care of with https://github.com/colindembovsky/cols-agent-tasks/commit/e1de3683fb3cb1d96d76f6268690146cff255a87

Can it be closed @colindembovsky?

**Comment by kspanier2013** [comment]
It looks like the warning in Azure DevOps stopped appearing sometime between 14 Feb 2025 and 14 March 2025. This doesn't look to be an issue anymore.

Tested with azure-hosted windows-2022 image, version 20250324.3.0, VersionAssemblies (2.2.178).
